jlibra-core/src/main/java/dev/jlibra/admissioncontrol/query/GetAccountState.java

```diff
@@ -2,8 +2,22 @@
 
 import org.immutables.value.Value;
 
+import com.google.protobuf.ByteString;
+
+import types.GetWithProof.GetAccountStateRequest;
+import types.GetWithProof.RequestItem;
+
 @Value.Immutable
 public interface GetAccountState {
 
     byte[] getAddress();
+
+    default RequestItem toGrpcObject() {
+        GetAccountStateRequest getAccountStateRequest = GetAccountStateRequest.newBuilder()
+                .setAddress(ByteString.copyFrom(this.getAddress()))
```

Comment:
Can we drop `this.`? Not only here, accross the PR

Reply:
:+1: